test/t8n/t8n.cpp

```diff
@@ -54,6 +55,13 @@ int main(int argc, const char* argv[])
             output_result_file = argv[i];
         else if (arg == "--output.alloc" && ++i < argc)
             output_alloc_file = argv[i];
+        else if (arg == "--state.reward" && ++i < argc)
+        {
+            if (std::string_view(argv[i]) == "-1")
+                block_reward = {};
```

Comment:
Do we actually need to set this here? Can we just have `!= "-1"` case for setting the value?